internal/controller/gitsync_controller.go

```diff
@@ -137,6 +138,12 @@ func (r *GitSyncReconciler) reconcile(ctx context.Context, gitSync *apiv1.GitSyn
 	if !gitSync.DeletionTimestamp.IsZero() {
 		logger.Infow("Deleting", "GitSync", gitSync)
 		if r.syncer != nil {
+			// Delete the linked resources to the GitSync
+			// Is gitSync.Name is the correct value for annotation ?
+			err := kubernetesshared.DeleteResourcesByAnnotations(ctx, r.client, common.PredefinedGroupVersionKinds, common.AnnotationKeyGitSyncInstance, gitSync.Name)
```

Comment:
At here and all other places we delete the resources, we should not returning the error if it is a `NotFound` error. This is stated in the issue description: "Note that if Not Found error occur during deleting the resources, it should be ignored."

Reply:
I am already ignoring that in here -https://github.com/numaproj-labs/numaplane/blob/deletGitsync/internal/util/kubernetes/kubernetes.go#L117

Reply:
I have made the changes  as per Syncer Cache so thats why its not working .Not relevant now .Thanks